en/docs/install-and-setup/setup/api-controller/managing-apis-api-products/importing-apis-via-dev-first-approach.md

```diff
@@ -385,6 +385,16 @@
             visibility: PUBLIC
     ```
 
+!!! note
+    For AI APIs, use the AnthropicAPI sample project as a reference and follow its folder and file structure:
+    [AnthropicAPI](https://github.com/wso2/product-apim-tooling/tree/master/import-export-cli/integration/testdata/AnthropicAPI)
```

Comment:
Should we link to the master branch here? Although this might work for now, when we have newer versions, this master may end up being incompatible with the 4.6.0 then. Shall we link to the branch like so:

https://github.com/wso2/product-apim-tooling/blob/4.6.x/import-export-cli/integration/testdata/...

Optionally, we can add this to samples-apim repo too